pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -64,6 +64,13 @@
     from pennylane.ops.op_math import Adjoint
     from pennylane.wires import Wires
 
+    # pylint: disable=import-error, no-name-in-module, ungrouped-imports
+    from pennylane_lightning.core._serialize import (
+        QuantumScriptSerializer,
+        global_phase_diagonal,
+    )
+    from pennylane_lightning.core._version import __version__
+
```

Comment:
Isn't the location of these headers changed in a different PR? I'm wondering if these are relocated again by isort or manually :thinking:

Reply:
`isort`'s doing.